script/vsts/upload-artifacts.js

```diff
@@ -93,6 +94,11 @@ async function uploadArtifacts () {
 
     console.log(`New release notes:\n\n${newReleaseNotes}`)
 
+    const releaseSha =
+        !isNightlyRelease
+            ? spawnSync('git', ['rev-parse', 'HEAD']).stdout.toString().trimEnd()
+            : undefined
```

Comment:
@daviwil: Can you help me understand why it's OK for the SHA to be undefined for nightly releases, but it's not OK for it to be undefined for stable and beta releases? Is it because the nightly releases should always use the default target commitish (i.e., `master`)?

If so, how would you feel about replacing `undefined` with `'master'` here?

Reply:
Good question!  Yep, Nightly should use the default which is `master`, I'll change it there to be more explicit.  The reason I'm using `master` for Nightly is because the tag actually gets created in the [atom/atom-nightly-releases](https://github.com/atom/atom-nightly-releases/tags) repo instead of the atom/atom repo, making the tag itself pretty meaningless.